www/source/index.html.slim

```diff
@@ -140,32 +143,32 @@ header.bg-gradient.margin-top-offset.short-bg.relative
 .row.strict-center.mobile-row
   .large-6.medium-6.columns.mobile-12.margin-under-xs
     .box-white.shadow.strict-center.fit-height.relative.slide-up
-      img.icon-art.margin-under-xs src="/images/home/platform.svg" onerror="this.src='/images/home/platform.png'" /
+      img.icon-art.margin-under-xs src="/images/home/platform.svg" onerror="this.src='/images/home/platform.png'" alt="image for InSpec platform support"/
       h4.t-purple Platform Agnostic
       p
-        |  InSpec supports all major operating systems and is platform agnostic, allowing you the freedom to run compliance assessments anywhere.
-      img.icon-logos src="/images/home/platform-logos.svg" onerror="this.src='/images/home/platform-logos.png'" /
+        |  InSpec supports all major operating systems and is platform agnostic, allowing you the freedom to run compliance and security tests anywhere.
+      img.icon-logos src="/images/home/platform-logos.svg" onerror="this.src='/images/home/platform-logos.png'" alt="logos for linux, windows, azure, mac, ubuntu, docker container, AWS, VMware"/
```

Comment:
We capitalize items in the list below in other places, should we here?

Example:
```
alt="logos for Linux, Windows, Azure, Mac, Ubuntu, Docker Container, AWS, VMware"
```